README.md

```diff
@@ -157,6 +157,47 @@ sparkEmrConfigs := Some(
 )
 ```
 
+## Use EmrBootstrapActionConfig to specify bootstrap actions
```

Comment:
Maybe we can use `BootstrapAction` instead of `EmrBootstrapActionConfig` to make our code shorter?